src/style/fonts.less

```diff
@@ -30,4 +30,9 @@
 	font-weight: normal;
 	font-size: 16px;
 	line-height: 22px;
-}
\ No newline at end of file
+}
+
+.div-section-larger-text {
```

Comment:
FYI, we have two pre-defined classes for text size: `.small` (which comes from Bootstrap) and `.large`

Reply:
Can we decide how small or how large .small and .large are?

Reply:
You could add a local widget-specific rule to define "small for this widget". The idea is to do that rarely. Hopefully, the site-wide css rules for .small and .large can be used, which makes for consistent typography across pages and widgets.